tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -1,50 +1,48 @@
 ---
-title: Secure Connections to Serverless Tier Clusters
-summary: Introduce TLS connection in TiDB Serverless Tier clusters.
+title: TLS Connection to Serverless Tier
+summary: Introduce TLS connection in TiDB Serverless Tier.
 ---
 
-# Secure Connections to Serverless Tier Clusters
+# TLS Connection to Serverless Tier
 
-This document introduces the core information about TLS in TiDB Serverless Tier.
+Establishing a secure TLS connection between your client and your TiDB Cloud Serverless Tier cluster is one of the basic security practices for connecting to your databases. The server certificate for Serverless Tier is issued by an independent third-party certificate provider [Let's Encrypt](https://letsencrypt.org/). Hence, you can easily connect to your Serverless Tier cluster without downloading a server-side digital certificate.
```

Comment:
There seems to be no obvious causal relationship between TiDB Cloud's use of the Let's Encrypt certificate and users not having to download the certificate. Because for the common user, they don't know what the Let's Encrypt certificate is.

Reply:
I think you are right ，for customer ，It do not need to descirb TLS from which the third partner CA ，what do you think about it @zhangyangyu

Reply:
I agree with @Daemonxiao . We could leave out some implementation details. Users just don't care about it.